org.eclipse.january/src/org/eclipse/january/dataset/IDataset.java

```diff
@@ -166,6 +166,7 @@
 	 * @return the metadata, may be null
```

Comment:
oops, missed this. Do you need to add the @deprecated javadoc marker, to point to the method that should be used instead?